Testing/xASL_ut_UnitTesting.m

```diff
@@ -68,6 +68,9 @@
     
     % Unit test: xASL_test_GetLogContent
     UnitTests(4) = xASL_ut_UnitTest_function_GetLogContent(TestRepository);
+    
+    % Unit test: xASL_test_GetLogContent
+    UnitTests(5) = xASL_ut_UnitTest_function_ExportVTK(TestRepository);
```

Comment:
just as a comment - you put here fixed numbers - UnitTests(5), which is of course totally fine. But what if the test-scripts are modified in several branches at once, then it can happen that both people add #5. shouldn't we use something like iTest and increment it at each application?

Reply:
Good idea, until now only I was writing those, but there are conflicts already xD

Reply:
Done in e8b7532